src/ert/scheduler/lsf_driver.py

```diff
@@ -165,7 +165,7 @@ def parse_bhist(bhist_output: str) -> Dict[str, Dict[str, int]]:
 class LsfDriver(Driver):
     def __init__(
         self,
-        exclude_hosts: Optional[Sequence[str]] = None,
+        exclude_hosts: Optional[str] = None,
```

Comment:
while at it, can you move this line to below resource_requirement and before bsub_cmd (some kind of artificial sorting by importance)